src/color.rs

```diff
@@ -1072,3 +1245,45 @@ where
 
     Ok(into_color(lightness, chroma, hue, alpha))
 }
+
+#[inline]
+fn parse_color_function<'i, 't, ComponentParser>(
+    component_parser: &ComponentParser,
+    arguments: &mut Parser<'i, 't>,
+) -> Result<Color, ParseError<'i, ComponentParser::Error>>
+where
+    ComponentParser: ColorComponentParser<'i>,
+{
+    let location = arguments.current_source_location();
+    let color_space = arguments.try_parse(|i| {
+        let ident = i.expect_ident()?;
+        PredefinedColorSpace::from_str(ident)
+            .map_err(|_| location.new_unexpected_token_error(Token::Ident(ident.clone())))
+    })?;
```

Comment:
I don't think you need the `try_parse` here, if you don't parse a correct color space you just return an error either way, so let's remove it.

Reply:
Nice catch, removing the `try_parse`